website/source/docs/job-specification/reschedule.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "docs"
+page_title: "reschedule Stanza - Job Specification"
+sidebar_current: "docs-job-specification-reschedule"
+description: |-
+  The "reschedule" stanza specifies the group's rescheduling strategy upon allocation failures.
+  The reschedule strategy can be configured with number of attempts and a time interval.
+  Nomad will only attempt to reschedule failed allocations on to another node only after
+  any local [restarts](docs/job-specification/restart.html) have been exceeded.
+---
+
+# `reschedule` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> **reschedule**</code>
+    </td>
+    <td>
+      <code>job -> group -> **reschedule**</code>
+    </td>
+  </tr>
+</table>
+
+The `reschedule` stanza specifies the group's rescheduling strategy.
+It can be configured with number of attempts and a time interval.
+If specified at the job level, the configuration will apply to all groups within the job.
+If multiple `reschedule` stanzas are specified, they are merged with the group stanza
```

Comment:
"If multiple" -> "If the reschedule stanza is present on both the job and the group, they are merged..."